checkmessages.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+#set -e
```

Comment:
Why deactivated?

Reply:
Checks are independent from each other and failing one of them does not necessary require to abort all subsequent checks.